pkg/syncthing/syncthing.go

```diff
@@ -362,6 +369,28 @@ func (s *Syncthing) WaitForPing(ctx context.Context, local bool) error {
 	}
 }
 
+// IdentifyReadinessIssue attempts to identify the issue that is preventing syncthing from being ready
+func (s *Syncthing) IdentifyReadinessIssue(fs afero.Fs) error {
+	if s.RegexMatchesLogs(fs, regexp.MustCompile("Error opening database: mkdir .*: no space left on device")) {
+		return oktetoErrors.ErrInsufficientSpace
```

Comment:
I would be more specific that the lack of space is in the developer's machine, not in the remote container

Reply:
agree, updated in 679f0d40 to be:

```
ErrInsufficientSpaceOnUserDisk = fmt.Errorf("there isn't enough disk space available on your machine to continue")
```